tests/model_serving/model_server/multi_node/conftest.py

```diff
@@ -96,3 +123,84 @@ def multi_node_predictor_pods_scope_class(
         client=admin_client,
         isvc=multi_node_inference_service,
     )
+
+
+@pytest.fixture(scope="function")
+def patched_multi_node_isvc_external_route(
+    multi_node_inference_service: InferenceService,
+) -> Generator[InferenceService, Any, Any]:
+    with ResourceEditor(
+        patches={
+            multi_node_inference_service: {
+                "metadata": {"labels": {Labels.Kserve.NETWORKING_KSERVE_IO: Labels.Kserve.EXPOSED}},
+            }
+        }
+    ):
+        for sample in TimeoutSampler(
+            wait_timeout=Timeout.TIMEOUT_1MIN,
+            sleep=1,
```

Comment:
polling every second for 1 minute seems a bit too much. Do we need to poll this often?

Reply:
this happens relatievyl quickly; do not want to wait